app/javascript/orangelight/availability.es6

```diff
@@ -59,11 +72,12 @@ export default class AvailabilityUpdater {
               if (allowRetry) {
                 console.log(`Retrying availability for record ${this.id}`);
                 window.setTimeout(() => {
+                  this.update_av_retrying();
                   this.request_availability(false);
                 }, 1500);
               } else {
                 console.error(`Failed to retrieve availability data for the bib (retry). Record ${this.id}: ${errorThrown}`);
-                this.update_single_undetermined();
+                this.update_av_undertermined();
               }
```

Comment:
can we rename `update_av_undetermined` -> `update_availability_undetermined` ? If yes, please apply the change in the rest of the code.

Reply:
Done.